wikipron/extract/default.py

```diff
@@ -16,15 +15,13 @@
 IPA_XPATH_SELECTOR = '//span[@class = "IPA"]'
 
 
-def _yield_phn(
-    request: requests.Response, config: "Config"
-) -> "Iterator[Pron]":
+def _yield_phn(request: requests_html, config: "Config") -> "Iterator[Pron]":
```

Comment:
I must say I'm surprised that `requests_html` is the correct type for our purposes here (I thought it's a module, given the way it's imported?). Thank you for figuring this out and helping us!